addons/website_slides/views/rating_rating_views.xml

```diff
@@ -86,6 +74,16 @@
         </field>
     </record>
 
+    <record id="rating_rating_view_tree_slides" model="ir.ui.view">
```

Comment:
What is the purpose of this view ? Without explicit priority set we cannot know if you want to do a default view or not ... and it does not seem to be used.

Reply:
The idea behind this view was to make a default view for the ratings without the comments. Is it relevant ? If so, should I change the base view tree instead of having an custom view ?

Reply:
Should it always be the default view ? Or only in slides ?

Reply:
Only in slides